openedx/core/djangoapps/programs/utils.py

```diff
@@ -120,7 +120,6 @@ def invert_programs(self):
                     program_list = inverted_programs[course_uuid]
                     if program not in program_list:
                         program_list.append(program)
-                    continue
```

Comment:
This is necessary because with this early exit from the continue, we weren't adding course run IDs to the inverted_programs dictionary (we only added course uuids).  We want both to be there for downstream users to look up either.